docs/release-notes.md

```diff
@@ -33,6 +33,86 @@ n8n uses [semantic versioning](https://semver.org/){:target=_blank .external-lin
 You can find the release notes for older versions of n8n [here](/release-notes/0-x)
 ///
 
+## n8n@1.64.3
+
+View the [commits](https://github.com/n8n-io/n8n/compare/n8n@1.64.2...n8n@1.64.3){:target=_blank .external-link} for this version.<br />
+**Release date:** 2024-10-25
+
+This release contains bug fixes.
+
+For full release details, refer to [Releases](https://github.com/n8n-io/n8n/releases){:target=_blank .external-link} on GitHub.
+
+## n8n@1.65.1
+
+View the [commits](https://github.com/n8n-io/n8n/compare/n8n@1.65.0...n8n@1.65.1){:target=_blank .external-link} for this version.<br />
+**Release date:** 2024-10-25
+
+This release contains bug fixes.
+
+For full release details, refer to [Releases](https://github.com/n8n-io/n8n/releases){:target=_blank .external-link} on GitHub.
+
+## n8n@1.65.0
+
+View the [commits](https://github.com/n8n-io/n8n/compare/n8n@1.64.1...n8n@1.65.0){:target=_blank .external-link} for this version.<br />
+**Release date:** 2024-10-24
+
+/// warning | [Breaking change](https://github.com/n8n-io/n8n/blob/master/packages/cli/BREAKING-CHANGES.md){:target=_blank .external-link}
+What changed?
+Queue polling via the environment variable `QUEUE_RECOVERY_INTERVAL` has been removed.
+
+When is action necessary?
+If you have set `QUEUE_RECOVERY_INTERVAL`, you can remove it as it no longer has any effect.
+///
+
+This release contains a new features, a new nodes, node enhancements, and bug fixes.
+
+<div class="n8n-new-features" markdown>
+
+### New node: n8n Form
+Use the [n8n Form node](/integrations/builtin/core-nodes/n8n-nodes-base.form/) to create user-facing forms with multiple pages. You can add other nodes with custom logic between to process user input. Start the workflow with a [n8n Form Trigger](/integrations/builtin/core-nodes/n8n-nodes-base.formtrigger/). 
+
+<figure markdown="span">
+    ![A multi-page form with branching](/_images/integrations/builtin/core-nodes/n8n-nodes-base.form/example_image.png)
+    <figcaption>A multi-page form with branching</figcaption>
+</figure>
+
+Additionally you can: 
+- Set default selections with query parameters  
```

Comment:
```suggestion

- Set default selections with query parameters  
```
Not sure if this will fix, but currently the first bullet is not being rendered correctly.